src/components/films/films.jsx

```diff
@@ -1,25 +1,29 @@
 import React from 'react';
 import { Row, Col } from 'reactstrap';
 
-import { films } from '../../constants';
+import { useFilms } from '../../hooks/use-films';
 import { CardFilm } from '../card-film/card-film';
 
-export const Films = () => (
-  <div className="list-films w-100">
-    <Row>
-      {films.map(({
-        name, type, id, src, cinemaId,
-      }) => (
-        <Col key={id}>
-          <CardFilm
-            name={name}
-            type={type}
-            src={src}
-            id={id}
-            cinemaId={cinemaId}
-          />
-        </Col>
-      ))}
-    </Row>
-  </div>
-);
+export const Films = () => {
+  const { films } = useFilms();
+
+  return (
+    <div className="list-films w-100">
+      <Row>
+        {films.map(({
+          name, type, id, src, cinema,
+        }) => (
+          <Col key={id}>
+            <CardFilm
+              name={name}
```

Comment:
this could be just a single property like **film** or **data**

Reply:
done